flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogTest.java

```diff
@@ -78,4 +80,17 @@ public void testCreateHiveTable() {
                 prop.keySet().stream()
                         .noneMatch(k -> k.startsWith(CatalogPropertiesUtil.FLINK_PROPERTY_PREFIX)));
     }
+
+    @Test
+    public void testRetrieveFlinkProperties() throws ClassNotFoundException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
```

Comment:
Can you choose another way to test this? Relying on private method is not a good practice.

Reply:
Thanks for you review, i change the test case, please review it.